README.md

```diff
@@ -1,6 +1,7 @@
 # rpmlint
 
 [![Build Status](https://travis-ci.org/rpm-software-management/rpmlint.svg)](https://travis-ci.org/rpm-software-management/rpmlint)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/3539541ffbaf442ab34a098d30af166f)](https://www.codacy.com/app/scarabeusiv/rpmlint?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=rpm-software-management/rpmlint&amp;utm_campaign=Badge_Grade)
```

Comment:
Is it not possible to enable the app under the organization rather than from you?

Reply:
@dmach Is this something we can do?